man/coap_handler.txt.in

```diff
@@ -207,6 +207,32 @@ COAP_EVENT_TCP_FAILED         0x1003
 COAP_EVENT_SESSION_CONNECTED  0x2001
 COAP_EVENT_SESSION_CLOSED     0x2002
 COAP_EVENT_SESSION_FAILED     0x2003
+/**
+ * (Q-)BLOCK receive errors
+ */
+COAP_EVENT_PARTIAL_BLOCK     = 0x3001,
```

Comment:
I suggest using the same syntax as above (i.e., no equal sign, no comma). Also for `COAP_EVENT_SERVER_SESSION_NEW` and `OAP_EVENT_SERVER_SESSION_DEL` below.